.changeset/pretty-houses-whisper.md

```diff
@@ -0,0 +1,6 @@
+---
+'@ai-sdk/provider-utils': patch
```

Comment:
Happy to change this, not sure if this is the correct approach for beta packages